### PR TITLE
Add tagged worker group with separate instance profile.

### PR DESCRIPTION
**Pull request by jmcarp**

### CODE REVIEW

File: concourse-production.yml [hunk 1]
```diff
@@ -28,3 +28,10 @@ instance_groups:
   azs: [z1]
   networks:
   - name: production-concourse
+- name: iaas-worker
+  vm_type: production-concourse-worker
+  vm_extensions: [production-concourse-iaas-profile]
+  instances: 1
```
Comment (cnelson) [inline concourse-production.yml]: I'm conflicted over running one of anything in production vs having two machines that will be idle 99% of the time.

Comment (cnelson) [inline concourse-production.yml]: I'm conflicted over running one of anything in production vs having two machines that will be idle 99% of the time.

Comment (cnelson) [inline concourse-production.yml]: IDK if we can go smaller than `t2.medium` with all the addons we pack onto those boxes.  `t2.small` only comes with 2GB ram.

Comment (jmcarp) [inline concourse-production.yml]: Looks like workers are `m3.xlarge`, so we can probably get away with `m3.large` or even `m3.medium`.

Comment (cnelson) [inline concourse-production.yml]: OIC, I was reading the cloud-config wrong and looking at the web nodes 😁 

In that case, let's go for 2x `m4.large` ?  It'll be cheaper than  1x `m3.xlarge`

**Pull request merged**